test/acceptance/wdio.conf.js

```diff
@@ -1,38 +1,69 @@
+var browserstack = require('browserstack-local')
```

Comment:
`var` should not be used.  Should be `const` or `let`.  As this value never changes, should be `const`